rm-datasource/src/main/java/io/seata/rm/datasource/exec/oracle/OracleJdbcType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.rm.datasource.exec.oracle;
+
+/**
+ * @author doubleDimple lovele.cn@gmail.com
+ */
+public class OracleJdbcType {
+
+    public static final int TIMESTAMP_WITH_TIME_ZONE = -101;
```

Comment:
Be careful not to duplicate the int value of java.sql.Types.

Reply:
The following are duplicate types:
![image](https://user-images.githubusercontent.com/96671314/202336881-0bc95fe8-fe61-4b70-b396-4170c1f8f598.png)